akka-http-core/src/main/scala/akka/http/impl/model/parser/SimpleHeaders.scala

```diff
@@ -226,6 +230,14 @@ private[parser] trait SimpleHeaders { this: Parser with CommonRules with CommonA
     rule { oneOrMore(addr).separatedBy(listSep) ~ EOI ~> (`X-Forwarded-For`(_)) }
   }
 
+  def `x-forwarded-host` = rule {
```

Comment:
Are there any specifications we could link to?

Reply:
More or less official, https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Proto and https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Host

The links are on the Java code, I can copy them over the Scala ones as well (I was following the existing pattern).

Reply:
Are these sources good enough? We could also link to the official `Forwarded` RFC that defines each sub-part of it (host, proto, for) https://tools.ietf.org/html/rfc7239#section-5.3 and https://tools.ietf.org/html/rfc7239#section-5.4